src/System.Linq.Parallel/src/System/Linq/Parallel/Utils/Sorting.cs

```diff
@@ -59,18 +59,18 @@ internal class SortHelper<TInputOutput, TKey> : SortHelper<TInputOutput>, IDispo
             OrdinalIndexState indexState, IComparer<TKey> keyComparer,
             GrowingArray<TKey>[] sharedkeys, TInputOutput[][] sharedValues, Barrier[][] sharedBarriers)
         {
-            Contract.Assert(source != null);
-            Contract.Assert(groupState != null);
-            Contract.Assert(sharedIndices != null);
-            Contract.Assert(sharedkeys != null);
-            Contract.Assert(sharedValues != null);
-            Contract.Assert(sharedBarriers != null);
-            Contract.Assert(sharedIndices.Length <= sharedkeys.Length);
-            Contract.Assert(sharedIndices.Length == sharedValues.Length);
+            Debug.Assert(source != null);
+            Debug.Assert(groupState != null);
+            Debug.Assert(sharedIndices != null);
+            Debug.Assert(sharedkeys != null);
+            Debug.Assert(sharedValues != null);
+            Debug.Assert(sharedBarriers != null);
+            Debug.Assert(sharedIndices.Length <= sharedkeys.Length);
+            Debug.Assert(sharedIndices.Length == sharedValues.Length);
             // Multi-dim arrays are simulated using jagged arrays.
             // Because of that, when phaseCount == 0, we end up with an empty sharedBarrier array.
             // Since there are no cases when phaseCount == 0 where we actually access the sharedBarriers, I am removing this check.
-            // Contract.Assert(sharedIndices.Length == sharedBarriers[0].Length);
```

Comment:
There are a handful of commented out asserts in these files.  Should we just delete them as part of this change?